tests/virt_autotest/login_console.pm

```diff
@@ -191,12 +201,14 @@ sub login_to_console {
         send_key 'ret';
     }
 
-    # Set ssh console timeout for thunderx machine
-    set_ssh_console_timeout_before_use if (is_remote_backend && is_aarch64 && get_var('IPMI_HW') eq 'thunderx');
+    # Set ssh console timeout for virt tests on ipmi backend machines
+    # it will make ssh serial console alive even with long time command
+    set_ssh_console_timeout_before_use if (is_remote_backend and is_x86_64 and get_var('VIRT_AUTOTEST', ''));
     # use console based on ssh to avoid unstable ipmi
     use_ssh_serial_console;
     # double-check xen role for xen host
     double_check_xen_role if (is_xen_host and !get_var('REBOOT_AFTER_UPGRADE'));
+    check_kvm_modules if is_kvm_host and !get_var('REBOOT_AFTER_UPGRADE');
```

Comment:
Is it ok for s390 or arm tests? @guoxuguang  @waynechen55

Reply:
ARM64 does not have KVM module because KVM module is eithe intel_kvm or amd_kvm I think.

Reply:
Ok, I'll skip arm and s390 machines.